pkg/kubectl/describe.go

```diff
@@ -1447,7 +1447,7 @@ type HorizontalPodAutoscalerDescriber struct {
 }
 
 func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string) (string, error) {
-	hpa, err := d.client.Extensions().HorizontalPodAutoscalers(namespace).Get(name)
```

Comment:
Not changing as explained in https://github.com/kubernetes/kubernetes/pull/20501#issuecomment-180260757